test/test_solid_errors.rb

```diff
@@ -6,4 +6,8 @@ class TestSolidErrors < Minitest::Test
   def test_that_it_has_a_version_number
     refute_nil ::SolidErrors::VERSION
   end
+
+  def test_default_base_controller
+    assert_equal ActionController::Base, SolidErrors::ApplicationController.superclass
+  end
 end
```

Comment:
Seems like we're only testing the default Superclass. we could also have test to assert that the custom configured controller is used as expected.